src/amberscript/parser.cc

```diff
@@ -304,6 +304,8 @@ Result Parser::Parse(const std::string& data) {
       r = ParseDeviceFeature();
     } else if (tok == "DEVICE_EXTENSION") {
       r = ParseDeviceExtension();
+    } else if (tok == "DEVICE_PROPERTY") {
```

Comment:
Can some tests be added for this parsing, similar to the `DEVICE_EXTENSIONS` tests in amberscript/parser_extension_test.cc ?

Reply:
Thank you. Will implement.